docs/docs/plugins-themes-and-starters.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Plugins, Themes, & Starters
+overview: true
+---
+
+In the Gatsby ecosystem, there's more than one way to build a site. To help you understand the differences between plugins, themes, and starters, this guide will talk through some of the details.
+
+## What is a plugin?
+
+Gatsby plugins are Node.js packages that implement Gatsby APIs and are commonly installed through a registry like npm. There are many types of [plugins](/plugins/), including data sourcing, SEO, responsive images, offline support, support for Sass, TypeScript, sitemaps and RSS, Google Analytics, and more. You can also [make your own plugins](/docs/creating-plugins/) and either distribute them for fellow Gatsby developers to use or [install them locally](/docs/loading-plugins-from-your-local-plugins-folder/).
+
+- [Plugin docs](/docs/plugins/)
+- [Using a plugin](/docs/using-a-plugin-in-your-site/)
+- [Plugin library](/plugins/)
+- [Creating plugins](/docs/creating-plugins/)
+
+## What is a theme?
+
+A Gatsby theme is a type of plugin that includes a `gatsby-config.js` file and adds pre-configured functionality, data sourcing, and/or UI code to Gatsby sites. Like plugins, themes can be packaged and distributed through a registry like npm or yarn, and versions can be updated through a site's `package.json` file.
```

Comment:
```suggestion
A Gatsby theme is a type of plugin that includes a `gatsby-config.js` file and adds pre-configured functionality, data sourcing, and/or UI code to Gatsby sites. Since they are plugins, themes can be packaged and distributed through a registry like npm or yarn, and versions can be updated through a site's `package.json` file.
```
Not quite sure how to phrase this, but wondering if "like a plugin" undercuts the message that themes are plugins?

Reply:
Works for me! I added those words because the first version of the text made it sound like themes could be distributed but plugins couldn't. This helps!